lib/ansible/modules/apt_repository.py

```diff
@@ -76,6 +76,12 @@
               Should usually only be set when working with a PPA on a non-Ubuntu target (e.g. Debian or Mint)
         type: str
         version_added: '2.3'
+    install_python_apt:
+        description:
+            - Whether to automatically install python apt or not, if it is not already installed.
+            - If python-apt Python library is not installed this module will not work.
```

Comment:
```suggestion
            - If python-apt Python library is not installed this module will not work.
            - Note that it is installed by running C(apt-get install python-apt) for Python 2, and C(apt-get install python3-apt) for Python 3.
              If you are using a Python on the remote that is not the system Python, this will not work. In that case,
              you should set I(install_python_apt=false) and need to ensure that the Python apt module for your
              Python version is installed in another way.
```